tools/configure_server_settings.rb

```diff
@@ -0,0 +1,37 @@
+if ARGV.empty?
+  puts "USAGE: #{__FILE__} server_id settings_path_separated_by_a_/ value [settings_path_separated_by_a_/ value]"
```

Comment:
I wonder if it would be easier if `settings_path_separated_by_a_/ value` was split up into `-p` and `-v` for a path and value.